PythonTool/WindowSettings.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from visit import *
+
+
+def WindowSettings(Shading=False):
```

Comment:
Each setting can be changed like the shading option. One just has to place each setting as an argument to the function. Then, modify all scripts that use WindowSettings.py accordingly.